mobileApp/app/src/main/java/com/lebel/novelbinge/MainActivity.kt

```diff
@@ -0,0 +1,34 @@
+package com.lebel.novelbinge
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.findNavController
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.navigateUp
+import androidx.navigation.ui.setupActionBarWithNavController
+import com.lebel.novelbinge.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var appBarConfiguration: AppBarConfiguration
+    private lateinit var binding: ActivityMainBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
Type hint `: Unit {`.

Reply:
nah ha.
![image](https://github.com/John-WL/story-book-generator/assets/12200284/fb1a88f2-7b64-4877-9a55-38923e9507f8)

Reply:
I would advise against relying on chatGPT for supporting your point of view. 
The `Unit` type is optional, so I won't fight for it! It's definitely not a bad practice to explicitely hint it. 
https://kotlinlang.org/docs/basic-syntax.html#variables
![image](https://github.com/John-WL/story-book-generator/assets/34081873/1f7024ba-f6bb-41df-b9c9-8a92c03c1cca)